packages/serial/src/ZWaveSocket.ts

```diff
@@ -18,7 +18,7 @@ export class ZWaveSocket extends ZWaveSerialPortBase {
 				open: (serial: net.Socket) =>
 					new Promise((resolve) => {
 						serial.on("close", () => {
-							if (this.isOpen) {
+							if (!this.isOpen) {
```

Comment:
Uhm, why? If the socket **is** open and suddenly out of nowhere the `"close"` event is emitted - that's unexpected IMO.

Reply:
Revert error, sorry